packages/katello/rubygem-katello/rubygem-katello.spec

```diff
@@ -43,8 +43,9 @@ Requires: %{?scl_prefix}rubygem(runcible) >= 2.11.0
 Requires: %{?scl_prefix}rubygem(runcible) < 3.0.0
 Requires: %{?scl_prefix}rubygem(anemone)
 Requires: %{?scl_prefix}rubygem(pulpcore_client)
-Requires: %{?scl_prefix}rubygem(pulp_ansible_client) < 1.0
-Requires: %{?scl_prefix}rubygem(pulp_file_client) < 1.0
+Requires: %{?scl_prefix}rubygem(pulp_file_client) < 3.0.0
```

Comment:
also here, sorting is weird. (but the version changes are correct)

Reply:
I think this one isn't sorted by the script and it just follows what's defined in the gemspec. If you look at the file, you'll see there's a lot of deps that aren't sorted. We can change the gem2rpm templates to sort.

Reply:
Right. wfm.